ManageWikiSettings.php

```diff
@@ -281,6 +281,19 @@
 		'help' => 'If enabled, moderators are allowed to edit pending changes before approving. DANGEROUS: moderator can accidentally delete the text of pending change. Enable this only when you use Moderation for pre-publish review.',
 		'requires' => [],
 	],
+	'wgModerationEmail' => [
+		'name' => 'Moderation Email',
+		'from' => 'moderation',
+		'type' => 'text',
+		'overridedefault' => $wgPasswordSender,
+		'section' => 'anti-spam',
+		'help' => 'If enabled, moderators are allowed to edit pending changes before approving. DANGEROUS: moderator can accidentally delete the text of pending change. Enable this only when you use Moderation for pre-publish review.',
```

Comment:
copy-paste error (this is description of the `$wgModerationEnableEditChange` above)

Reply:
Fixed that. Thanks!